src/model/errors.rs

```diff
@@ -0,0 +1,21 @@
+use std::fmt;
+
+#[derive(Debug)]
+pub struct RobotparserError {
```

Comment:
I'd call it `Error`, and it needs to implement `std::error::Error`

Reply:
I've implemented first but I saw that the Error description is deprecated cf https://github.com/rust-lang/rust/pull/66919 do you think we should still implement it ?

Reply:
I think you can just add a blank impl for it

Reply:
Ok thanks its done in b2525b5a